app/views/vignettes/slides/_form_question.html.erb

```diff
@@ -0,0 +1,52 @@
+<%= form.fields_for :question do |question_form| %>
+  <div>
+    <%= question_form.label :type, "Question Type" %>
+    <%= question_form.select :type, options_for_select([
```

Comment:
Bug: assume a slide which has a text field answer is saved. Now edit that slide and choose "No question". Then _take_ the slide as a user: you will still see the text that was stored in the text field beforehand.